rotkehlchen/exchanges/okx.py

```diff
@@ -50,6 +50,8 @@
 logger = logging.getLogger(__name__)
 log = RotkehlchenLogsAdapter(logger)
 
+OKX_ENDPOINTS: TypeAlias = Literal['currencies', 'trading balance', 'funding balance', 'trades', 'deposits', 'withdrawals']  # noqa: E501
+
```

Comment:
Would it make sense to convert this to an enum and use the endpoint as value?. Since you compare against it I believe it is better and less error prone